pkg/util/billing/billing_test.go

```diff
@@ -135,22 +140,22 @@ func TestDelete(t *testing.T) {
 			name: "successful mark for deletion on billing entity, with a subscription not registered for e2e",
 			fixture: func(f *testdatabase.Fixture) {
 				f.AddBillingDocuments(&api.BillingDocument{
-					Key:                       strings.ToLower(testdatabase.GetResourcePath(mockSubID, "resourceName")),
-					ClusterResourceGroupIDKey: fmt.Sprintf("/subscriptions/%s/resourcegroups/resourceGroup", mockSubID),
-					ID:                        mockSubID,
+					Key:                       strings.ToLower(testdatabase.GetResourcePath(subID, "resourceName")),
+					ClusterResourceGroupIDKey: fmt.Sprintf("/subscriptions/%s/resourcegroups/resourceGroup", subID),
+					ID:                        docID,
```

Comment:
Just want to confirm that the `mockSubID` --> `docID` changes here and elsewhere are intended. They seem to have different values. Does that also mean the test will pass regardless of the value of `ID`?

Reply:
They are intended, yes.  Somewhere else in the diff there's another equivalent mockSubID -> docID change.  The goal is that different IDs should be used for the fields with different meanings.